lib/jmespath/nodes/function.rb

```diff
@@ -34,6 +36,12 @@ def initialize(name)
 
       private
 
+      def error_raise(error_type, message)
```

Comment:
I would actually suggest something along the lines of `maybe_raise`, `potential_raise`, or `ignorable_raise`.